dev/js/reducers/commons/document.js

```diff
@@ -0,0 +1,26 @@
+import {
+  GET_USER_DOCUMENTS,
+  UPLOAD_USER_DOCUMENT,
+  UPLOAD_USER_DOCUMENT_SUCCESS,
+  UPLOAD_USER_DOCUMENT_FAIL,
+  GET_DOC_TYPES,
+} from '../../constants/types';
+
+const defaultState = {
+  userDocuments: [],
+  docTypes: [],
+};
+
+export default function (state = defaultState, action) {
+  switch (action.type) {
+    case GET_USER_DOCUMENTS: {
+      let newState = { ...state };
+      return newState;
+    }
+    case GET_DOC_TYPES: {
+      let newState = { ...state };
+      return newState;
```

Comment:
bro u didn't do anything in these reducers, u just return the same state ,so why dispatch action, if there is no need then remove the dispatch from action.